src/components/NcEmojiPicker/NcEmojiPicker.vue

```diff
@@ -109,12 +109,27 @@ This component allows the user to pick an emoji.
 			:show-preview="showPreview"
 			:title="previewFallbackTitle"
 			v-bind="$attrs"
-			@select="select" />
+			@select="select">
+			<template #searchTemplate="slotProps">
+				<NcTextField ref="search"
+					class="search"
+					:value.sync="search"
+					:label="t('Search')"
+					:label-visible="true"
+					:placeholder="slotProps.i18n.search"
```

Comment:
do you actually need to use `slotProps` ?
```suggestion
					:placeholder="i18n.search"
```

Reply:
Good point

Now using `i18n` directly while keeping the `slotProps.onSearch()` which is necessary for emoji search https://github.com/serebrov/emoji-mart-vue/blob/a8ac66cffd2bd08f64371add84ea71b7ef620dca/src/utils/picker.js#L97-L104